sageresearch-mpower/src/main/java/org/sagebionetworks/research/mpower/sageresearch/ui/WebConsentFragment.kt

```diff
@@ -130,10 +143,24 @@ class WebConsentFragment : DaggerFragment() {
                 SharingScope.fromValue(jsonObject.getString("scope")))
     }
 
+    fun onWebViewLoading() {
+        consent_webview.visibility = INVISIBLE
+    }
+
+    fun onWebViewLoadingFinished() {
+        consent_webview.visibility = VISIBLE
+    }
+
     fun onWebViewReceiveError(url: String?, errorCode: Int, description: String?) {
         LOGGER.warn("WebView received error: {} with code: {} for request: {}", description, errorCode, url)
     }
 
+    fun onConsentLoading() {
+    }
+
+    fun onConsentError() {

```

Comment:
Display error to the user?

Reply:
This currently has a TODO in iOS also. I'll remove these methods which aren't called so it's not confusing.